samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -25,12 +25,20 @@ NET_BUF_POOL_FIXED_DEFINE(tx_pool, 1, CONFIG_BT_ISO_TX_MTU, NULL);
 static struct bt_conn *default_conn;
 static struct bt_audio_stream streams[MAX_PAC];
 
+
+static uint8_t unicast_server_addata[] = {
+	0x4E, 0x18, /* ASCS UUID */
+	0x01, /* Target Announcement */
+	0xFF, 0x00, 0xFF, 0x00, /* Available Audio Context */
+	0x00, /* Metadata length */
+};
+
 /* TODO: Expand with BAP data */
 static const struct bt_data ad[] = {
-	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
-	BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_ASCS_VAL)),
```

Comment:
We should still advertise the `BT_DATA_UUID16_ALL` type with `BT_UUID_16_ENCODE(BT_UUID_ASCS_VAL)`, even if we advertise the service data as well.

Reply:
> BT_DATA_UUID16_ALL

I don't think so. BT_DATA_UUID_ALL is 0x03, «Complete List of 16-bit Service Class UUIDs». However, BAP asks «Service Data - 16-bit UUID» which is 0x16. They are different.

Reply:
Yes, but from 8.1.1.1 in BAP V 1.0, we also have 
![image](https://user-images.githubusercontent.com/2520413/147551521-965a6d45-fba8-446d-91de-80c7f6bc3ceb.png)


So we should both advertise the flags, the service UUID AD data and the service data itself. It could of course be argued that if we advertise the service data from ASCS, that we could omit ASCS from the list of Service UUIDs.

Reply:
> Yes, but from 8.1.1.1 in BAP V 1.0, we also have ![image](https://user-images.githubusercontent.com/2520413/147551521-965a6d45-fba8-446d-91de-80c7f6bc3ceb.png)
> 
> So we should both advertise the flags, the service UUID AD data and the service data itself. It could of course be argued that if we advertise the service data from ASCS, that we could omit ASCS from the list of Service UUIDs.
<img width="1033" alt="Screen Shot 2021-12-28 at 10 32 30 PM" src="https://user-images.githubusercontent.com/4544551/147576744-894d8565-6c0e-4f9f-bcaa-24bbd69b24d6.png">
This is the AUX_ADV_IND which I'm trying to implement, I believe its Advertising Data Field is compatible with BAP definition, isn't it？ :)

Reply:
@kaiser-ren The flags looks correct, but it doesn't have the Service UUID AD data type as I pointed out. See https://www.bluetooth.com/specifications/specs/core-specification-supplement-10/ section 1.1

Reply:
> @kaiser-ren The flags looks correct, but it doesn't have the Service UUID AD data type as I pointed out. See https://www.bluetooth.com/specifications/specs/core-specification-supplement-10/ section 1.1

I see, it looks like below.
Frame 22: 75 bytes on wire (600 bits), 75 bytes captured (600 bits) on interface /dev/cu.usbmodem14201-3.6, id 0
nRF Sniffer for Bluetooth LE
Bluetooth Low Energy Link Layer
    Access Address: 0x8e89bed6
    Packet Header: 0x3147 (PDU Type: AUX_ADV_IND, TxAdd: Random)
        .... 0111 = PDU Type: 0x7 AUX_ADV_IND
        ...0 .... = Reserved: 0
        ..0. .... = Reserved: 0
        .1.. .... = Tx Address: Random
        0... .... = Reserved: 0
        Length: 49
    Extended Advertising Header
        ..00 1001 = Extended Header Length: 9
        01.. .... = Advertising Mode: Connectable Non-scannable (0x1)
        Extended Header Flags: 0x09
        Advertising Address: d8:69:b1:a6:83:6e (d8:69:b1:a6:83:6e)
        Advertiser Data Info: 0x0e15
    Advertising Data
        Flags
            Length: 2
            Type: Flags (0x01)
            000. .... = Reserved: 0x0
            ...0 .... = Simultaneous LE and BR/EDR to Same Device Capable (Host): false (0x0)
            .... 0... = Simultaneous LE and BR/EDR to Same Device Capable (Controller): false (0x0)
            .... .1.. = BR/EDR Not Supported: true (0x1)
            .... ..1. = LE General Discoverable Mode: true (0x1)
            .... ...0 = LE Limited Discoverable Mode: false (0x0)
        16-bit Service Class UUIDs
            Length: 3
            Type: 16-bit Service Class UUIDs (0x03)
            UUID 16: Unknown (0x184e)
        Service Data - 16 bit UUID
            Length: 9
            Type: Service Data - 16 bit UUID (0x16)
            UUID 16: Unknown (0x184e)
            Service Data: 011f000f0000
        Device Name: Unicast Audio Server
            Length: 21
            Type: Device Name (0x09)
            Device Name: Unicast Audio Server
    CRC: 0x141247